data/json/mapgen/militia/GM_LODGE.json

```diff
@@ -0,0 +1,291 @@
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
+        "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
+      ],
+      "palettes": [ "GMLODGEPALETTE" ],
+      "terrain": { ",": "t_region_groundcover" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "weight": 1000,
+    "om_terrain": [
+      [ "GMLODGE2_1", "GMLODGE2_2", "GMLODGE2_3", "GMLODGE2_4" ],
+      [ "GMLODGE3_1", "GMLODGE3_2", "GMLODGE3_3", "GMLODGE3_4" ]
+    ],
+    "object": {
+      "fill_ter": "t_floor",
+      "faction_owner": [
+        { "id": "GM_militia_faction", "x": [ 0, 23 ], "y": [ 0, 23 ] },
+        { "id": "GM_militia_faction", "x": [ 24, 47 ], "y": [ 0, 23 ] },
+        { "id": "GM_militia_faction", "x": [ 48, 71 ], "y": [ 0, 23 ] },
+        { "id": "GM_militia_faction", "x": [ 72, 95 ], "y": [ 0, 23 ] },
+        { "id": "GM_militia_faction", "x": [ 0, 23 ], "y": [ 24, 47 ] },
+        { "id": "GM_militia_faction", "x": [ 24, 47 ], "y": [ 24, 47 ] },
+        { "id": "GM_militia_faction", "x": [ 48, 71 ], "y": [ 24, 47 ] },
+        { "id": "GM_militia_faction", "x": [ 72, 95 ], "y": [ 24, 47 ] }
+      ],
+      "rows": [
```

Comment:
I'm not going to scour it with a comb but if you seem to have `t_floor` under at least most of the furniture in the palette already to deal with the upper floor so you could remove the `fill_ter` and make the ` `s `.`s and `C`s ` `s if you wanted to make it less harsh on the eye. For future reference using `#` for walls makes windows and doors stand out more too.

Reply:
I've made most of the changes you've recommended but this one I'm having trouble interpreting. I have switched out Walls to # and cupboards to "u" instead but the "so you could remove the fill_ter and make the  s .s and Cs  s if you wanted to make it less harsh on the eye. " i'm not sure how to interpret. Sorry.

Reply:
I wouldn't worry about it it's not a big deal

Reply:
This is going to sound really stupid but as I was explaining it to a friend I just realized what you meant. 🤣